### PR TITLE
Added message system with pusher

### PR DESCRIPTION
**Pull request by Dagniele**
Message system

### CODE REVIEW

File: frontend/src/components/Match/Spectate.vue [hunk 1]
```diff
@@ -137,6 +168,8 @@ import VuejsDialog from 'vuejs-dialog'
 import 'vuejs-dialog/dist/vuejs-dialog.min.css'
 import io from 'socket.io-client'
 
+window.Pusher.logToConsole = true
```
Comment (Dagniele) [inline frontend/src/components/Match/Spectate.vue]: Shit, please remove this, it is just for dev

File: frontend/src/components/Player/Leaders.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@
             </span>
             <div style="margin-top: 10px;">
               <div v-for="(leader, index) in filteredAllTimePointLeaders" v-bind:key="leader.id" class="row-item">
-                <router-link :to="'/player/' + leader.id + '/info'">{{ index+1 }}. {{ leader.name }}</router-link>
```
Comment (miclud) [inline frontend/src/components/Player/Leaders.vue]: Please don't revert my fantastic feature :pray:

Comment (Dagniele) [reply frontend/src/components/Player/Leaders.vue]: that's just the diff, I didn't touch this file at all :smile:

Comment (Chmieloo) [reply frontend/src/components/Player/Leaders.vue]: @Dagniele Can you merge current master to your repository?

**Comment by Dagniele** [comment]
Yep

**Pull request closed without merging**